.github/workflows/python-app.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
This workflow fails. You should check out how to connect the Redis service container in order to make these tests succeed. Otherwise, just comment out the Redis test in the tests (but document this in the PR so that you make the users/dev team aware of this).

https://docs.github.com/en/actions/using-containerized-services/creating-redis-service-containers